3_task/binarySearch.js

```diff
@@ -0,0 +1,27 @@
+function binarySearch(originalArray, number) {
+  let array = originalArray;
+  let left = 0;
+  let right = array.length - 1;
+  let middle;
+  while((left != right)) {
```

Comment:
looks like infinite loop when array is []
`let right = array.length - 1;` is -1
-1 != 0

also if array is [-10] console.log(binarySearch(array, -10)) is undef